src/NuGet.Core/NuGet.Protocol/Model/LocalPackageSearchMetadata.cs

```diff
@@ -50,7 +50,7 @@ public IReadOnlyList<string> OwnersList
             {
                 if (_ownersList is null)
                 {
-                    _ownersList = Owners != null ? Owners.Split(',').Select(s => s.Trim()).ToList() : null;
+                    _ownersList = Owners != null ? Owners.Split(',').Where(s => !string.IsNullOrWhiteSpace(s)).Select(s => s.Trim()).ToList() : null;
```

Comment:
You can have `string.Split` not generate the emprty elements in the first place: https://learn.microsoft.com/en-us/dotnet/api/system.string.split?view=net-8.0#system-string-split(system-char()-system-stringsplitoptions)